### PR TITLE
`workdir`: Improve scripts and `README.md`

### PR DESCRIPTION
**Pull request by razvand**
Improve usability in scripts according to verbal feedback. Also improve `README.md` accordingly.

### CODE REVIEW

File: workdir/README.md [hunk 1]
```diff
@@ -44,54 +186,54 @@ In case library or application repositories have been updated upstream and you r
...
-
-So, after deciding to use a given set or predefined PRs, switch to the corresponding branch and then run the `fill-workdir.sh` script to populate the repositories with the `pr_list.inc` files.
-
-## Preparing the Build Environment
+The `pr_list.inc` file will define a `pr_list` Bash variable filled with the PR ids that you require.
 
-Inside each application directory, use the `make-hierarchy.sh` script to prepare the directory structure:
+You would then run the script:
 
 ```
```
Comment (StefanJum) [inline workdir/README.md]: Add `console` here.

Comment (StefanJum) [inline workdir/README.md]:
```suggestion
```text
```

File: workdir/README.md [hunk 2]
```diff
@@ -24,6 +77,95 @@ Files in `copy/common/` care duplicated in almost all repositories.
...
+```
+
+Select one of them and obtain the corresponding Unikraft image.
+
+**Note**: Currently, KraftKit doesn't support 9pfs.
+When using Kraftkit, aim for the `-initrd` targets.
+
+## Running Applications
+
+### Make-based Builds
```
Comment (StefanJum) [inline workdir/README.md:146]:
```suggestion
### Make-based Applications
```

File: workdir/README.md [hunk 3]
```diff
@@ -24,6 +77,95 @@ Files in `copy/common/` care duplicated in almost all repositories.
...
+run-fc-x86_64-initrd.sh  run-qemu-aarch64-9pfs.sh  run-qemu-aarch64-initrd.sh  run-qemu-x86_64-9pfs.sh  run-qemu-x86_64-initrd.sh
+```
+
+For example, to run the QEMU image for x86_64 using `initrd` as a filesystem, use:
+
+```console
+./run-qemu-x86_64-initrd.sh
+```
+
+### KraftKit-based Builds
```
Comment (StefanJum) [inline workdir/README.md:161]:
```suggestion
### KraftKit-based Applications
```

File: workdir/README.md [hunk 4]
```diff
@@ -24,6 +77,95 @@ Files in `copy/common/` care duplicated in almost all repositories.
 In `copy/base/` you have a specific file per directory, and the copying is something like `cp -r copy/base/* .`.
 In `copy/common/` the files are common to all directories, and the copying is something like `for d in apps/*/; do cp copy/common/* "$d"; done`.
```
Comment (StefanJum) [inline workdir/README.md:78]: Add the `copy/common-nocopy/` directory here.

**Review by StefanJum — CHANGES_REQUESTED** [review]

**Review by StefanJum — APPROVED** [review]
Looks fine to me, thank you.

**Review by gabrielmocanu — APPROVED** [review]
LGTM

**Pull request merged**